cgltf.h

```diff
@@ -1722,7 +1718,8 @@ cgltf_result cgltf_validate(cgltf_data* data)
 
 cgltf_result cgltf_copy_extras_json(const cgltf_data* data, const cgltf_extras* extras, char* dest, cgltf_size* dest_size)
```

Comment:
Alternatively we could leave this function as is in the meantime, and simply mark it as deprecated - that way we can remove this together with start_offset / end_offset?